### DISCUSSION TITLE
Adding new institution choices

### DISCUSSION THREAD

**Discussion by cuttle-fish-my** [OpenSIST/OpenSIST.github.io#23] [Ideas]
Hey guys, we notice that some users request to add a `customization` option for the `university name` selection on the `add program` page. However, letting users freely type the institution (university/laboratory) name may lead to potential redundancy. Therefore, to solve this problem, we manually added the institution request by users.

To request a new institution, please follow the following template:

Institution name: 

Institution abbreviation:

Institution website:

**Comment by Black2222White** [comment]
Institution name: The Rockefeller University

Institution abbreviation: RU

Institution website: https://www.rockefeller.edu/

**Reply by cuttle-fish-my** [reply]
Hi, Black2222White! Before adding your own application records, please make sure the according program has been added :)
<img width="1416" height="1349" alt="image" src="https://github.com/user-attachments/assets/46b1d212-9bba-43c2-8e69-777c856473f0" />
